node_exporter/linux/ccp_backrest_version.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+###
+#
+# Copyright 2017-2021 Crunchy Data Solutions, Inc. All Rights Reserved.
+#
+###
+
+# Additional metric option to monitor disk queue depth with node_exporter. Set the DISK variable to the disk system that is to be monitored
+
+OUTPUTDIR=/var/lib/ccp_monitoring/node_exporter
+VERTMP=backrest_version.tmp
+VERFILE=backrest_version.prom
+if [ ! -d ${OUTPUTDIR} ]; then
+  mkdir ${OUTPUTDIR}
+fi
```

Comment:
I personally like `mkdir -p`, but that's a personal style thing.

Reply:
Done. Package should technically take care of this, so I'm debating even leaving this in here. Was carryover from the other script I adapted this from.